core-bundle/src/Resources/contao/library/Contao/Feed.php

```diff
@@ -197,7 +197,7 @@ public function generateAtom()
 		{
 			$xml .= '<entry>';
 			$xml .= '<title>' . StringUtil::specialchars(strip_tags(StringUtil::stripInsertTags($objItem->title))) . '</title>';
-			$xml .= '<content type="xhtml"><div xmlns="http://www.w3.org/1999/xhtml">' . preg_replace('/[\n\r]+/', ' ', StringUtil::toXhtml($objItem->description)) . '</div></content>';
+			$xml .= '<content type="html">' . preg_replace('/[\n\r]+/', ' ', StringUtil::specialchars($objItem->description)) . '</content>';
```

Comment:
```suggestion
			$xml .= '<content type="html">' . preg_replace('/[\n\r]+/', ' ', htmlspecialchars($objItem->description, ENT_XML1, 'UTF-8')) . '</content>';
```

We need to use entity encoding for XML here I think.

We need to enable